src/templates/admin/elements/generate_form.html

```diff
@@ -4,8 +4,8 @@
 <div class="form-group">
     <div class="{% if field.field.widget.attrs.div_class %}{{ field.field.widget.attrs.div_class }}{% else %}large-12 columns{% endif %}">
         {% if field.field.widget.attrs.is_checkbox %}
-        {% if field.errors %}<span class="red">{{ field.errors|join:", " }}</span><br/>{% endif %}
-        {{ field }} &nbsp;{% if field.help_text %}{{ field.help_text|safe }}{% endif %}
+          {% if field.errors %}<span class="red">{{ field.errors|join:", " }}</span><br/>{% endif %}
+          {{ field }} {{ field.label }}&nbsp;{% if field.help_text %}<small>{{ field.help_text|safe }}</small>{% endif %}
```

Comment:
In the user interface for custom review fields, this puts out HTML like this:

```html
<div class="form-group">
    <div class="large-12 columns">
          <input type="checkbox" name="3" is_checkbox="" id="id_3"> Test element&nbsp;
    </div>
</div>
```

That's not [how HTML labels should be marked up](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/label). There needs to be a `label` element and it should be linked to the `input` id with the `for` attribute. This will make the label clickable in place of the checkbox to change the selection, and it will make the whole thing more accessible in other ways as well.